cli/cmd/tapRunner.go

```diff
@@ -497,7 +498,8 @@ func watchApiServerPod(ctx context.Context, kubernetesProvider *kubernetes.Provi
 				isPodReady = true
 				go startProxyReportErrorIfAny(kubernetesProvider, cancel)
 
-				if err := apiserver.Provider.InitAndTestConnection(GetApiServerUrl(), 20); err != nil {
+				retries := config.GetEnvConfig(reflect.Int, config.ApiServerRetries, reflect.ValueOf(20))
```

Comment:
it is something that should be inside the provider as its internal configuration

Reply:
i am not sure i understood what you meant